controllers/automation/packagedeployment_controller.go

```diff
@@ -173,17 +184,28 @@ func (r *PackageDeploymentReconciler) applyPackageMutations(ctx context.Context,
 
 	r.l.Info("parsed resources", "resources", pkgBuf)
 
-	// search for NS and ScaleProfile resources
 	nsIdx := -1
-	scaleProfileIdx := -1
+	bindingResources := make(map[yaml.ResourceIdentifier]*yaml.RNode)
 	for i, n := range pkgBuf.Nodes {
+		// search for namespace
 		if n.GetApiVersion() == "v1" && n.GetKind() == "Namespace" {
 			nsIdx = i
-			continue
 		}
-		if n.GetApiVersion() == "infra.nephio.org/v1alpha1" && n.GetKind() == "ClusterScaleProfile" {
-			scaleProfileIdx = i
-			continue
+
+		// search for binding resources
+		annotations := n.GetAnnotations()
+		if value, found := annotations["config.kubernetes.io/local-config"]; found && value == "binding" {
```

Comment:
Nice. For "Day 2" (later PR): add watches for these types, so we can update the PackageRevision when they change.